src/components/SidebarMenuItem/SidebarMenuItem.js

```diff
@@ -50,6 +55,7 @@ const defaultProps = {
   className: 'tm-sidebar-item',
   tabIndex: 0,
   accessKey: undefined,
+  body: undefined,
```

Comment:
body  seems very similar to children property , why not using the existing `children` property to define the content of the SideBarMenuItem ?

Reply:
Makes total sense, just updated and pushed